### PR TITLE
Fix #2020

### PR DESCRIPTION
**Pull request by hntvy96**
This is
✔ Bug Fix

See #41.

### CODE REVIEW

File: src/test/java/com/codecentric/sample/store/service/CustomerServiceTest.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public Customer answer(InvocationOnMock invocation) throws Throwable {
             }
         }).when(hostService).expand(any(Customer.class));
 
-        when(addressService.getPLZForCustomer(customer)).thenReturn(478913);
```
Comment (phucnh4196) [inline src/test/java/com/codecentric/sample/store/service/CustomerServiceTest.java]: ???

**Review by phucnh4196 — COMMENTED** [review]
Please fix this

**Review by phucnh4196 — COMMENTED** [review]
Ok

**Comment by hntvy96** [comment]
Ok

**Review by phucnh4196 — COMMENTED** [review]
123

**Review by phucnh4196 — COMMENTED** [review]
1

**Comment by phucnh4196** [comment]
Close #41

**Pull request merged**